safe/gui/tools/peta_bencana_dialog.py

```diff
@@ -554,3 +593,53 @@ def download(self, url, output_path):
                 self,
                 self.tr('Download error'),
                 self.tr(message))
+
+    # The function below might be usefull for future usage.
+
+    # def get_available_area(self):
+    #     """Function to automatically get the available area on API.
+    #        *still cannot get string data from QByteArray*
+    #     """
+    #     available_area = []
+    #     network_manager = QNetworkAccessManager()
+    #     api_url = QUrl('https://data.petabencana.id/cities')
+    #     api_request = QNetworkRequest(api_url)
+    #     api_response = network_manager.get(api_request)
+    #     data = api_response.readAll()
+    #     json_response = QScriptEngine().evaluate(data)
+    #     geometries = json_response.property('output').property('geometries')
+    #     iterator = QScriptValueIterator(geometries)
+    #     while iterator.hasNext():
+    #         iterator.next()
+    #         geometry = iterator.value()
+    #         geometry_code = (
+    #             geometry.property('properties').property('code').toString())
+    #         available_area.append(geometry_code)
+
+    def populate_combo_box(self):
+        """Populate combo_box."""
+        if self.radio_button_production.isChecked():
+            self.source = production_api['url']
+            available_data = production_api['available_data']
+            self.combo_box.clear()
```

Comment:
Can you give a better name to this combobox ? like adding `city` for instance. We try to give explicit name.